vmm/modules/trusty_tee/trusty_tee.c

```diff
@@ -177,18 +204,17 @@ void init_trusty_tee(evmm_desc_t *evmm_desc)
 	if (trusty_desc->tee_file.barrier_size == 0)
 		print_warn("No rawhammer mitigation for TEE\n");
 
-	trusty_cfg.tee_runtime_addr = trusty_desc->tee_file.runtime_addr
-		- trusty_desc->tee_file.barrier_size;
-	trusty_cfg.tee_runtime_size = trusty_desc->tee_file.runtime_total_size
-		+ 2 * trusty_desc->tee_file.barrier_size;
+	runtime_addr = trusty_desc->tee_file.runtime_addr;
+	runtime_size = trusty_desc->tee_file.runtime_total_size;
+	trusty_cfg.tee_runtime_addr = runtime_addr;
+	trusty_cfg.tee_runtime_size = runtime_size;
```

Comment:
add barrier into calculation.
trusty_cfg.runtime addr/size will be used to set EPT, which should include the barrier

Reply:
add barrier_size